appveyor.yml

```diff
@@ -28,9 +28,9 @@ init:
 install:
   - cd c:\
   - appveyor DownloadFile http://windows.php.net/downloads/releases/archives/php-5.6.0-nts-Win32-VC11-x86.zip -FileName php.zip
-  - 7z x php.zip -oc:\php
+  - 7z x php.zip -oc:\php > nul
   - appveyor DownloadFile https://dl.dropboxusercontent.com/s/euip490d9183jkr/SQLSRV32.cab -FileName sqlsrv.cab
-  - 7z x sqlsrv.cab -oc:\php\ext php*_56_nts.dll
+  - 7z x sqlsrv.cab -oc:\php\ext php*_56_nts.dll > nul
```

Comment:
Yes `nul` is correct. And `$null` in Powershell  form what I have seen. At least here it works and hide the output of the unzipping operation